### ISSUE TITLE
Eye flashing damage is not consistent between tajara/vulp and moths

### ISSUE THREAD

**Issue by Miraviel** [ParadiseSS13/Paradise#18246]
**Issue Description**:
Creatures that can see in the dark (e.g. tajara, vulpkanin) get more damage from flashes because of their `see_in_dark` var. Tajara, by default, has a `2` for this var. If they are colourblind too, their `see_in_dark` variable becomes `8`.

In both cases, they received `3` damage from a flash, flashed them about 20 times on my test server. The code suggests that bigger darksight = more damage, but this is either not the case or it is but very rarely (and my RNG sucked).

According to the code (which I do not properly understand, so this report may be a dud), this is chance based. In this case, this is not in consistent with how moth eye damage is calculated, which is simply being `flash_protect = FLASH_PROTECTION_SENSITIVE` on their `/obj/item/organ/internal/eyes/nian`, which means they get a flat bonus to their eye damage every single time.

**What did you expect to happen**: 

Eye damage from flashes should be consistent on all races if it is their racial disadvantage - it should be either chance based or flat for both darksight races and moths.

**What happened instead**:

Tajara (and possibly vulp) damage seems to be chance based, moths' seem to be a flat bonus.

**Why is this bad/What are the consequences**:

I am looking into race balance (which is impossible to have, I know, but I wish to make it better) and this seems to be inconsistent. If one race has a "more prone to flashes" disadvantage, then that disadvantage should be in line with another race's "more prone to flashes" disadvantage. 

**Steps to reproduce the problem**:
1. Spawn two tajaran, one moth
2. Set one tajara to colourblind
3. Flash all three repeatedly
4. Check eye damage.

**When did the problem start happening**:
Unsure.

**Extra information**:
- Moth eyes: `modules\surgery\organs\subtypes\moth.dm`
- Tajara eyes: `modules\surgery\organs\subtypes\tajaran.dm`
- Eye damage calculation: `modules\mob\living\carbon\carbon.dm`
- Eye damage calculation 2: `modules\mob\living\carbon\human\human.dm`
- Flash item: `game\objects\items\devices\flash.dm`

**Comment by S34NW** [comment]
moths also take damage from flash sources when others would be protected.

This was an intentional design choice made when balancing moths.

**Comment by DGamerL** [comment]
Heya! After a bit of digging, the code that made Vulp/Taj eyes' darksight no longer be affected by colorblindness was merged a while ago in #8468 ! This would make sense as to why your test scenarios show the absolute same for both colorblind and not colorblind vulps/taj!
I've also looked a bit at the code and it seems that as of the current master branch, flash damage is consistently handled for any mob with darksight more than 2!
Thanks for making this report!
